src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -94,7 +100,7 @@ export interface SavedObjectsRepositoryOptions {
 /**
  * @public
  */
-export interface SavedObjectsIncrementCounterOptions extends SavedObjectsBaseOptions {
+export interface SavedObjectsIncrementCounterOptions<T = unknown> extends SavedObjectsBaseOptions {
```

Comment:
Super-NIT: I think the recommendation now is to avoid `T` and use descriptive names in the generics